code/modules/hydroponics/grown/root.dm

```diff
@@ -28,10 +28,15 @@
 /obj/item/food/grown/carrot/attackby(obj/item/I, mob/user, params)
 	if(I.get_sharpness())
 		to_chat(user, span_notice("You sharpen the carrot into a shiv with [I]."))
```

Comment:
The chat message should change depending on whether you form a sword or shiv.

Reply:
done